### ISSUE TITLE
Accessibility Markbot issue

### ISSUE THREAD

**Issue by lexyfh** [lexyfh/hear-me-oh-content#1]
<img width="833" alt="Screen Shot 2022-02-09 at 9 15 36 PM" src="https://user-images.githubusercontent.com/84150431/153324236-7378ee23-f8ed-47ca-b958-d2e2e303f20e.png">

Hey, I'm not sure where I went wrong.
@wendywarren

**Comment by wendywarren** [comment]
Hi @lexyfh,

I think what Markbot is referring to are the Read More links that are missing an aria-label.

When links aren't descriptive as to where the link will take a user we have to include aria-labels for those using screen readers.

Ex. `<a href="https://en.wikipedia.org/wiki/Sloth" aria-label="Read more about sloths">Read more</a>`

Let me know if that helps once you fix both Read more links.

Thanks,
Wendy

**Comment by lexyfh** [comment]
Okay I just tried doing that, also tried adding the aria label to the navigation but markbot still gives me the same issue.

@wendywarren

**Comment by wendywarren** [comment]
I downloaded your repo and added the aria-label to these lines and then the error went away after I refreshed Markbot.

`<a href="https://en.wikipedia.org/wiki/Sloth" aria-label="Read more about sloths">Read more</a>`

`<a href="https://en.wikipedia.org/wiki/Three-toed-sloth" aria-label="Read more about three-toed sloths">Read more</a>`

Is it giving a different error now?

**Comment by lexyfh** [comment]
That fixed it! I guess I didn't need the aria label in the footer. Thank you!
@wendywarren
